ui/lib/kv/addon/components/page/secret/details.hbs

```diff
@@ -19,7 +19,7 @@
   </:toolbarFilters>
   <:toolbarActions>
     {{#if @secret.canReadMetadata}}
-      <KvVersionDropdown @secret={{@secret}} @metadata={{@metadata}} @onClose={{this.onClose}} />
+      <KvVersionDropdown @currentVersion={{@secret.version}} @metadata={{@metadata}} @onClose={{this.onClose}} />
```

Comment:
nit: `currentVersion` makes me think that this is the current secret version (from `metadata.current_version` when this is just the version selected/viewed, right? 

thoughts on renaming to `secretVersion` or `displayVersion`?